lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -1221,10 +1221,14 @@ struct SwiftASTContextError : public llvm::ErrorInfo<SwiftASTContextError> {
 /// This indicates an error in the SwiftASTContext.
 struct ModuleImportError : public llvm::ErrorInfo<ModuleImportError> {
   static char ID;
-  std::string Message;
+  std::string msg;
+  bool is_explicit;
 
-  ModuleImportError(llvm::Twine Message) : Message(Message.str()) {}
-  void log(llvm::raw_ostream &OS) const override { OS << "ModuleImport"; }
+  ModuleImportError(llvm::Twine message, bool is_explicit = false)
+      : msg(message.str()), is_explicit(is_explicit) {}
+  void log(llvm::raw_ostream &OS) const override {
+    OS << (is_explicit ? "ex" : "im") << "plicit module import error: " << msg;
```

Comment:
If it's all the same, could these strings be separate? As is, if someone gets either of these messages, and tries to grep for (say) "explicit module import error", they won't find this line.